subsys/bluetooth/host/conn.c

```diff
@@ -287,6 +287,26 @@ static void conn_le_update_timeout(struct k_work *work)
 	atomic_set_bit(conn->flags, BT_CONN_SLAVE_PARAM_UPDATE);
 }
 
+static void tx_free(struct bt_conn_tx *tx)
+{
+	tx->conn = NULL;
```

Comment:
This should be:
```
if (tx->conn) {
        bt_conn_unref(tx->conn);
        tx->conn = NULL;
}
```
see my other comment about where the `ref()` is needed

Reply:
Fixed.